docs/any-api/api-reference.md

```diff
@@ -176,6 +216,65 @@ function requestPriceFrom(address _oracle)
 }
 ```
 
+### sendOperatorRequest
+
+> 📘 This function is similar to `sendChainlinkRequest`[function](#sendchainlinkrequest).
+> It is recommended to use `sendOperatorRequest` but make sure the oracle you are contacting is an `Operator` [contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/Operator.sol).
+
+```solidity
+function sendOperatorRequest(
+    Chainlink.Request memory _req,
+    uint256 _payment
+) returns (bytes32 requestId)
+```
+
+Sends the request payload to the stored oracle address. It takes a [Chainlink.Request](#chainlinkrequest) and the amount of LINK to send amount as parameters. The request is serialized and calls `operatorRequest` on the address stored in `chainlinkOracleAddress` via the LINK token's `transferAndCall` method.
+
+`sendOperatorRequest` returns the ID of the request. If your application needs to, your contract can store that ID, but you don't need to. The ChainlinkClient helpers will store the ID under the hood, along with the oracle address, and use them when you call `recordChainlinkFulfillment` in your callback function to make sure only that the address you want can call your Chainlink callback function.
```

Comment:
```suggestion
`sendOperatorRequest` returns the ID of the request. Optionally, your contract can store the ID if your application needs it. The `ChainlinkClient` helpers store the ID and the oracle address and use them when you call `recordChainlinkFulfillment` in your callback function. This ensures that only the specified address can call your Chainlink callback function.
```
The last sentence wasn't clear to me, so I tried rewriting it. Is my edit accurate?